### PR TITLE
ADEM-5982 Fix calculating preferred video size and prevent content jumping

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/video/player/ui/dynamicReveal.js [hunk 1]
```diff
@@ -20,7 +20,9 @@ define('ext.wikia.adEngine.video.player.ui.dynamicReveal', [
 				if (!slotExpanded) {
 					slotTweaker.expand(params.slotName);
 					slotExpanded = true;
-					video.ima.dispatchEvent('wikiaSlotExpanded');
+					setTimeout(function () {
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/js/video/player/ui/dynamicReveal.js]: Can we add few words why do we need to delay this operation?

File: extensions/wikia/AdEngine/js/video/player/porvata/floaterConfiguration.js [hunk 2]
```diff
@@ -43,7 +40,9 @@ define('ext.wikia.adEngine.video.player.porvata.floaterConfiguration', [
 			onBeforeDetach: function (floatingContext) {
 				var viewport = floatingContext.elements.viewport;
 
-				viewport.style.height = viewport.offsetHeight + 'px';
+				// Magic 14 is related to #INCONTENT_WRAPPER > #INCONTENT_PLAYER margins
+				// and prevents jumping when video is going to float
+				viewport.style.height = (viewport.offsetHeight - 14) + 'px';
```
Comment (kvas-damian) [inline extensions/wikia/AdEngine/js/video/player/porvata/floaterConfiguration.js:45]: Maybe we could calculate `14` based on DOM node properties (margins in this case)?

Comment (fraszczakszymon) [reply extensions/wikia/AdEngine/js/video/player/porvata/floaterConfiguration.js:45]: It would require using `getComputedStyle` - is it worth?

Comment (kvas-damian) [reply extensions/wikia/AdEngine/js/video/player/porvata/floaterConfiguration.js:45]: Nein! Let's leave it as it is. Comment is enough.

**Review by kvas-damian — APPROVED** [review]

**Review by nandy-andy — APPROVED** [review]

**Pull request merged**